spec/lib/generators/forem/dummy/templates/config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
I don't think we should include this by default. SimpleForm already defaults to its own default configuration if one isn't specified. Why do we need to specify it ourselves?